src/elements/tooltip/tooltip.spec.ts

```diff
@@ -284,4 +284,54 @@ describe('sbb-tooltip', () => {
       expect(tooltipOutlet!.children[0].localName).to.equal('sbb-tooltip');
     });
   });
+
+  describe('delay attributes', () => {
```

Comment:
```suggestion
  describe('attribute usages - delay attributes', () => {
```

Good set of test, i would just change the name